Mobile/stockdogmobile/App.js

```diff
@@ -20,8 +20,7 @@ export default class App extends Component {
 
   render() {
     return(
-      this.state.fontLoaded ? 
-        Routes() : null
+      this.state.fontLoaded ? Routes() : null
```

Comment:
Right now when it waits for the fonts to load it doesn't break, but I think you're right. What kind of thing do you think I should put if the fonts never load?

Reply:
I think a little error message would do better than a blank screen (I'm assuming that is what happens if the font doesn't load.) This way the user wouldn't be waiting thinking the app is just hanging.

Reply:
I just tried implementing the error page, and now whenever it loads the app it displays that error message for a split second while the state is still changing as the font loads. Since we don't want this right now, I think we should just keep the null there. But I agree that I should display something prettier than that red error page if the font doesn't load, so I can look into that in the next sprint!